src/app.js

```diff
@@ -15,6 +15,8 @@ const urlJoin = require('url-join');
 
 const config = require('./config');
 const conversationLogger = require('./conversationLogger');
+const dispatcher = require('./dispatcher');
```

Comment:
Why is there a dispatcher _and_ a router? This seems like something that will cause a lot of confusion.